datanode/gateway/graphql/schema.graphql

```diff
@@ -3790,8 +3787,6 @@ type NewMarket {
   liquidityMonitoringParameters: LiquidityMonitoringParameters!
   "Decimal places for order sizes, sets what size the smallest order / position on the market can be"
   positionDecimalPlaces: Int!
-  "Liquidity Provision order price range"
-  lpPriceRange: String!
```

Comment:
This shouldn't be removed so that it can still be compatible rather than a breaking change.